packages/blocks/src/__internal__/utils/block-range.ts

```diff
@@ -335,7 +336,8 @@ type ExperimentBlockRange = {
   endModel: BaseBlockModel;
   betweenModels: BaseBlockModel[];
   startOffset: number;
-  endOffset: number;
+  // undefined for infinite until the block end
+  endOffset: number | undefined;
```

Comment:
I think `BlockRange` is deprecated, but will it better to use `Math.Infinity` instead of `undefined` here?

Reply:
emm, sounds better